test/testdata/workspaces/typescript-sample-app-lint-error/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import express = require("express");
```

Comment:
maybe we could remove express here to speed things up? A simple HTTP server should suffice here, right?